platform/android/export/export.cpp

```diff
@@ -2562,106 +2632,42 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
 			CLEANUP_AND_RETURN(ERR_SKIP);
 		}
 		Error err = OK;
-		Vector<String> cl = cmdline.strip_edges().split(" ");
-		for (int i = 0; i < cl.size(); i++) {
-			if (cl[i].strip_edges().length() == 0) {
-				cl.remove(i);
-				i--;
-			}
-		}
-
-		gen_export_flags(cl, p_flags);
 
 		if (p_flags & DEBUG_FLAG_DUMB_CLIENT) {
-
 			APKExportData ed;
 			ed.ep = &ep;
 			ed.apk = unaligned_apk;
 			err = export_project_files(p_preset, ignore_apk_file, &ed, save_apk_so);
 		} else {
-			//all files
-
-			if (apk_expansion) {
-
-				String apkfname = "main." + itos(version_code) + "." + get_package_name(package_name) + ".obb";
-				String fullpath = p_path.get_base_dir().plus_file(apkfname);
-				err = save_pack(p_preset, fullpath);
-
-				if (err != OK) {
-					unzClose(pkg);
-					EditorNode::add_io_error("Could not write expansion package file: " + apkfname);
-
-					CLEANUP_AND_RETURN(ERR_SKIP);
-				}
-
-				cl.push_back("--use_apk_expansion");
-				cl.push_back("--apk_expansion_md5");
-				cl.push_back(FileAccess::get_md5(fullpath));
-				cl.push_back("--apk_expansion_key");
-				cl.push_back(apk_expansion_pkey.strip_edges());
-
-			} else {
-
-				APKExportData ed;
-				ed.ep = &ep;
-				ed.apk = unaligned_apk;
-
-				err = export_project_files(p_preset, save_apk_file, &ed, save_apk_so);
-			}
-		}
```

Comment:
It does seem like you're changing the logic here.  The new logic in the `else` branch now occurs all the time, whereas previously it only occurred when `apk_expansion` was false.
It that on purpose?

Reply:
I'll change the `else` to be `else if (!apk_expansion)`. The logic change was not on purpose.

Reply:
I think I addressed the feedback you had for this PR, can you check?